README.md

```diff
@@ -167,6 +168,31 @@ expect(descendant).not.toContainElement(ancestor)
 // ...
 ```
 
+### `toContainHTML`
+
+```typescript
+toContainHTML(htmlText: string)
+```
+
+Assert whether a string representing a HTML element is contained in another element:
+
+```javascript
+// add the custom expect matchers once
+import 'jest-dom/extend-expect'
+
+// ...
+// <span data-testid="parent"><span data-testid="child"></span></span>
+const parent = queryByTestId('parent')
+expect(parentElement).toContainHTML('<span data-testid="child"></span>')
+// ...
+```
+
+> Chances are you probably do not need to use this matcher. We encourage testing from the perspective of how the user perceives the app in a browser. That's why testing against a specific DOM structure is not advised.
+
```

Comment:
It's important to keep the `>` in front of these empty lines, so it all appears as a single note. The same applies to the next empty line below.